### PR TITLE
Slideshow - Add support for texture selections

### PR DESCRIPTION
**Pull request by TyroneMF**
**When merged this pull request will:**
- Adds support for texture selections of objects instead of only using 0.

### CODE REVIEW

**Review by jonpas — APPROVED** [review]

File: addons/slideshow/functions/fnc_addSlideActions.sqf [hunk 1]
```diff
@@ -19,7 +20,7 @@
  * Public: No
  */
 
-params ["_objects", "_images", "_names", "_controller", "_currentSlideshow"];
+params ["_objects", "_images", "_names", "_controller", "_currentSlideshow", "_selection"];
```
Comment (veteran29) [inline addons/slideshow/functions/fnc_addSlideActions.sqf:23]: Description says it defaults to 0.
```suggestion
params ["_objects", "_images", "_names", "_controller", "_currentSlideshow", ["_selection", 0]];
```

**Review by veteran29 — APPROVED** [review]

**Comment by TyroneMF** [comment]
@Dahlgren pboProject build failing on things i didn't touch.

**Comment by PabstMirror** [comment]
>Missing File Summary
ace_spottingscope.p3d : \a3\data_f\penetration\metal.rvmat
ace_spottingscope.p3d : \a3\data_f\penetration\plastic.rvmat

**Review by PabstMirror — APPROVED** [review]

**Comment by Dahlgren** [comment]
@TyroneMF fixed, some issue with the injected a3 data

File: addons/slideshow/stringtable.xml [hunk 2]
```diff
@@ -227,6 +227,12 @@
             <Chinese>每張幻燈片顯示的時間。 預設:0 (自動換圖已禁用)</Chinese>
             <Chinesesimp>每张幻灯片显示的时间。 预设:0 (自动换图已禁用)</Chinesesimp>
         </Key>
+        <Key ID="STR_ACE_Slideshow_Selection_DisplayName">
+            <English>Texture Selection</English>
+        </Key>
+        <Key ID="STR_ACE_Slideshow_Selection_Description">
+            <English>Object Texture selection. Default: 0</English>
```
Comment (mharis001) [inline addons/slideshow/stringtable.xml:234]:
```suggestion
            <English>Object texture selection. Default: 0</English>
```

**Review by mharis001 — APPROVED** [review]

**Pull request merged**